examples/shared_samples/hypervisor_cluster_profile.rb

```diff
@@ -0,0 +1,71 @@
+# (C) Copyright 2018 Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+
+require_relative '../_client' # Gives access to @client
+
+# Example: Create/Update/Delete hypervisor cluster profile
+# NOTE: This will create a hypervisor cluster profile named 'cluster5', update it and then delete it.
+#
+# Supported APIs:
+# - API800 for C7000
+# - API800 for Synergy
+
+# Resources that can be created according to parameters:
+# api_version = 800 & variant = C7000 to OneviewSDK::API800::C7000::HypervisorClusterProfile
+# api_version = 800 & variant = Synergy to OneviewSDK::API800::Synergy::HypervisorClusterProfile
+
+# Resource Class used in this sample
+hypervisor_cluster_profile_class = OneviewSDK.resource_named('HypervisorClusterProfile', @client.api_version)
+
+options = {
+  type: @hypervisor_type,
+  name: @hypervisor_cluster_profile_name,
+  hypervisorManagerUri: @hypervisor_manager_uri,
```

Comment:
I would ignore this for the time being as we are giving the values explicitly for other resources as well.
If you feel it is mandatory, then I'll do it once the branches are merged so that i have access to the hypervisor manager code.